superset-frontend/packages/superset-ui-chart-controls/src/types.ts

```diff
@@ -340,11 +341,24 @@ export interface ControlPanelSectionConfig {
   controlSetRows: ControlSetRow[];
 }
 
+export interface SharedFormData {
+  metrics: AdhocMetric[];
+  columns: (AdhocColumn | PhysicalColumn)[];
+}
+
+export interface iStandardizedFormData {
```

Comment:
We don't use "i" prefix for interface anywhere else in the codebase. Maybe we should rename it and stick to PascalCase?

Reply:
Good catch. the `i` is for the `interface` from some other language styling. I will update it.